Hi3Helper.Core/Lang/en.json

```diff
@@ -127,9 +127,9 @@
     "ListCol4": "Size",
     "ListCol5": "L.CRC",
     "ListCol6": "R.CRC",
-    "Status1": "Click Check Files to run the analysis",
-    "Status2": "Loading Indexes...",
-    "Status3": "{0} file(s) have been found to be broken ({1} in total size). Click Repair Game to begin the repair process",
+    "Status1": "Click 'Check Files' to integrity validation",
+    "Status2": "Loading Indices...",
```

Comment:
`Indexes` seems more fitting and its more widely known (either still right in terms of plural form) so leaving this should be fine.
https://www.nasdaq.com/articles/indexes-or-indices-whats-the-deal-2016-05-12

Reply:
Reverted changes